cmd/stake-dist/main.go

```diff
@@ -2,24 +2,43 @@ package main
 
 import (
 	"fmt"
-	"os"
 	"sort"
 	"strconv"
 
 	gaia "github.com/cosmos/cosmos-sdk/cmd/gaia/app"
 	"github.com/interchainio/delegation/pkg"
 	tmclient "github.com/tendermint/tendermint/rpc/client"
+
+	"github.com/spf13/cobra"
 )
 
 var (
 	cdc = gaia.MakeCodec()
 
-	// expects a locally running node
-	node = tmclient.NewHTTP("localhost:26657", "/websocket")
+	fullNodeUrl string
```

Comment:
Rather use the fully capitalized version of the variable name, like so:
```suggestion
	fullNodeURL string
```